doc/source/new/fqllogging.rst

```diff
@@ -239,74 +234,186 @@ The syntax of the nodetool enablefullquerylog command is as follows:
    -u <username>, --username <username>
   Remote jmx agent username
 
-Run the following command on each node in the cluster.
+To enable FQL, run the following command on each node in the cluster on which you want to enable logging:
 
 ::
 
  nodetool enablefullquerylog --path /tmp/cassandrafullquerylog
 
-After the full query logging has been  enabled run some CQL statements to generate full query logs.
+Disabling or resetting FQL
+-------------
+
+Use the ``nodetool disablefullquerylog`` to disable logging. 
+Use ``nodetool resetfullquerylog`` to stop FQL and clear the log files in the configured directory.
+**IMPORTANT:** Using ``nodetool resetfullquerylog`` will delete the log files! Do not use this command unless you need to delete all log files.
+
+fqltool
+-------
+
+The ``fqltool`` command is used to view (dump), replay, or compare logs.
+``fqltool dump`` converts the binary log files into human-readable format; only the log directory must be supplied as a command-line option.
+
+``fqltool replay`` (CASSANDRA-14618) enables replay of logs. 
+The command can run from a different machine or cluster for testing, debugging, or performance benchmarking. 
+The command, run on the same node on which the logs are generated can recreate a dropped database object.
```

Comment:
> The command, run on the same node on which the logs are generated can recreate a dropped database object

I don't know what this means, what are you trying to say?

Reply:
I didn't write that text originally. I've revised to: The command can also be used to recreate a dropped database object.